### PR TITLE
Fix case distDir is an absolute path

### PR DESCRIPTION
**Pull request by amiel**
Before, if `distDir` was an absolute path, `absoluteInputPath` would end up having rootPath doubled.  This uses `path.resolve` instead of `path.join` to resolve that issue.

### CODE REVIEW

File: tests/unit/index-nodetest.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+/* globals require, describe, before, beforeEach, it, process */
```
Comment (amiel) [inline tests/unit/index-nodetest.js]: Sorry about the incidental change here. Adding this calmed down my editor's jshinting. I'd be happy to remove it if undesired.

Comment (lukemelia) [inline tests/unit/index-nodetest.js]: Yes, let's at least save it for separate PR.

**Comment by lukemelia** [comment]
Besides comment, LGTM

**Comment by amiel** [comment]
@lukemelia thanks for the speedy review. I've removed the `globals` addition.

**Comment by lukemelia** [comment]
Thanks @amiel! :performing_arts:

**Comment by amiel** [comment]
Thanks @lukemelia. Can I get a release for this one?

**Pull request merged**